src/cancellable-fetch.ts

```diff
@@ -31,10 +31,15 @@ export const cancellableFetch = <T>(
 ): CancellableFetch<T> => {
   const abortController = new AbortController();
   const abort = () => abortController.abort();
+  const bearerToken = process.env.REACT_BEARER_TOKEN;
 
   const fetchPromise = fetch(url, {
     ...init,
-    headers: { ...init?.headers, Accept: 'application/json' },
+    headers: {
```

Comment:
I think all these request headers should be defined when `cancellableFetch` is called (in `checkAuthorization`), so that we can keep `cancellableFetch` more generic.

Reply:
Added this so that we can test it locally, instead of adding it on all the calls.